spec/javascripts/app/helpers/text_formatter_spec.js

```diff
@@ -216,7 +225,7 @@ describe("app.helpers.textFormatter", function(){
       });
 
       it("correctly encodes image src to punycode", function() {
-        _.each(this.evilUrls, function(url, num) {
+        _.each(this.evilUrls, function(url, num) {nintendo 
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.